src/helpers.js

```diff
@@ -162,35 +180,27 @@ function registerWith(Handlebars) {
         }
 
         if (typeof message === 'string') {
+            if (!(message instanceof SafeString)) {
+                message = escape(message);
```

Comment:
Err, this would always be truthy.  Fixing now.  Essentially need to escape message whenever any of the hash values is a safestring but the message itself was not passed in as a safestring (the changed we talked about this morning to knock out the boolean arg)